pkgs/applications/networking/p2p/lokinet/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, curl
+, libevent
+, libsodium
+, libuv
+, pkgconfig
+, sqlite
+, unbound
+, writeShellScriptBin
+, zeromq
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lokinet";
+  version = "0.9.9";
+  githash = "440b547d2c3e7384ec82ec21f0b1635e42bcda64";
+
+  src = fetchFromGitHub {
+    owner = "oxen-io";
+    repo = "lokinet";
+    rev = "v${version}";
+    sha256 = "AaGsRg9S9Cng9emI/mN09QSOIRbE+x3916clWAwLnRs=";
+    fetchSubmodules = true;
```

Comment:
```suggestion
```
We want to use our libraries.

Reply:
Great, another project hard vendoring libraries. If you don't know cmake well enough than we probably need to live with this for now.